dbt_subprojects/dex/models/aggregator_trades/dex_aggregator_trades.sql

```diff
@@ -59,7 +58,7 @@ SELECT
     , evt_index
 FROM {{ aggregator_model }}
 {% if is_incremental() %}
-WHERE block_date >= date_trunc('day', now() - interval '7' day)
+WHERE block_date >= date_trunc('day', now() - interval '7' day) and amount_usd < 1000000000
```

Comment:
added filter for trades over 1000000000, i.e. those that fail test  dbt_utils_accepted_range_dex_aggregator_trades_amount_usd__1000000000

Reply:
i have fixed this issue in prod, we should be fine without forcing the filter now

Reply:
removed filter